src/components/Accordions/Accordions.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react"
+import { useColorMode, useColorModeValue } from "@chakra-ui/react"
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from "@chakra-ui/react"
+
+const Accordions = () => {
+  const { toggleColorMode } = useColorMode()
+  const primary = useColorModeValue("#8282FF", "#FFE3D3")
```

Comment:
@TylerAPfledderer 

No worries. 

This is my first time contributing so it may take some time to understand all the logics. Hope you don't mind.

I will read the extra information you provided and let you know when I am done.

Reply:
@justynleung no problem here! 😄  Me and @pettinarip are certainly aware of some documentation that can be created to further support contributing while maintaining consistency in the codebase and the desired workflow. 

In the meantime, you are always welcome to reach out for any clarification!